x-pack/plugins/stack_connectors/common/crowdstrike/schema.ts

```diff
@@ -235,6 +235,7 @@ export const CrowdstrikeHostActionsParamsSchema = schema.object({
   actionParameters: schema.maybe(schema.object({}, { unknowns: 'allow' })),
   ids: schema.arrayOf(schema.string()),
   alertIds: schema.maybe(schema.arrayOf(schema.string())),
+  comment: schema.maybe(schema.string()),
```

Comment:
Is this subaction connector used "live" via the actionsClient, or can it be queued to be run as part of an alert action?  The distinction is that if it's an alert action, we have to follow the ["intermediate releases"](https://docs.google.com/document/d/1mU5jlIfCKyXdDPtEzAz1xTpFXFCWxqdO5ldYRVO_hgM/edit?usp=sharing) model of making changes to schema like this.

Reply:
Hey @pmuellr , thanks for checking up on this :) 
Currently we are only using this connector through `execute`, it cannot be added to `actions` through the rule. 
In the near future we are hoping to include this execute in the `response actions` that will be called from the rule. There is no plan to migrate to `system actions` yet. 
I hope this answers your question :)

Reply:
I guess it depends on when "near future" is, but seems likely it would not be used as a queued action in the time frame that an "intermediate release" would be required.  Please make sure you read that linked doc above ^^^.  When it becomes possible to run the connector as a rule action, changes like this would **require** an intermediate release ...

Reply:
Heh, unless I missed it, `comment` isn't even referenced in this PR - which is basically how you'd do an intermediate release anyway - or at least one way.